l10n_ch_import_cresus/wizard/l10n_ch_import_cresus.py

```diff
@@ -0,0 +1,344 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author Vincent Renaville
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import sys
+import traceback
+import logging
+import base64
+import csv
+import tempfile
+from openerp import models, fields, api
+from openerp.tools.translate import _
+from itertools import izip_longest
+from datetime import datetime
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountCresusImport(models.TransientModel):
+    _name = 'account.cresus.import'
+    _description = 'Export Accounting'
+
+    company_id = fields.Many2one('res.company', 'Company',
+                                      invisible=True)
+    period_id = fields.Many2one('account.period', 'Period',
+                                     required=True)
+    report = fields.Text(
+            'Report',
+            readonly=True
+        )
+    journal_id = fields.Many2one('account.journal', 'Journal',
+                                      required=True)
+    state = fields.Char(size=64,
+                        sting="Import state"
+                        'Report',
+                        readonly=True
+                        )
+    file = fields.Binary(
+            'File',
+            required=True
+        )
+    help_html = fields.Html('Import help', readonly=True)
+
+    _defaults = {'state': 'draft',
```

Comment:
default is now an attribute of fields.

https://www.odoo.com/documentation/8.0/reference/orm.html#basic-fields